packages/material-ui/src/Box/Box.test.js

```diff
@@ -57,4 +57,58 @@ describe('<Box />', () => {
     expect(element.getAttribute('font-family')).to.equal(null);
     expect(element.getAttribute('font-size')).to.equal(null);
   });
+
+  it('respect properties order when generating the CSS', function test() {
+    const isMozilla = window.navigator.userAgent.indexOf('Firefox') > -1;
+    const isJSDOM = /jsdom/.test(window.navigator.userAgent);
+
+    if (isJSDOM || isMozilla) {
+      // Test fails on Mozilla with just:
+
+      // "border": "",
+      // "border-color": "",
```

Comment:
Why is this intentional? Sounds like an actual bug to me.

Reply:
I’ve tested the scenario on the actual browser and it worked. I tried testing both border and border color, I will check if maybe borderTop and the other side properties are set.